assets/scss/layouts/_header.scss

```diff
@@ -487,4 +483,18 @@ button[data-link="home"]::before {
 [data-dark-mode] [data-bs-target="#offcanvasDoks"]:hover,
 [data-dark-mode] [data-bs-target="#offcanvasDoks"]:focus {
   border: 1px solid #495057 !important;
+}
```

Comment:
Weirdly, this file seems to be missing some table highlight lines, initially added here: https://github.com/CoherentLabs/cohtml/pull/1827/files
The PR is merged, but the contents seem to be missing from all documentations. Can you please add the same changes here, so that they are present in the merged docs

Reply:
Added! Thanks for finding this one I missed!